README.md

```diff
@@ -80,30 +120,60 @@ Install dependencies by executing the following command:
 $ npm install
 ```
 
+### Azure connection
+
+The AI features of the bot are powered by a number of Azure services. To connect to these services you will need to authenticate with Azure.
+If you are in the `DTS Platform Operations` Security Group you will have the permissions needed to use the services.
+
+To authenticate with Azure, run the following command:
+
+```bash
+az login
+```
+
+[See more information on authenticating with Azure in a local development environment](https://learn.microsoft.com/en-us/javascript/api/overview/azure/identity-readme?view=azure-node-latest#authenticate-the-client-in-development-environment).
+
+If you aren't in the `DTS Platform Operations group` you will need at least the following permissions on their respective resources:
```

Comment:
Would a non-platops person build this? (suppose a few teams do have a similar bot)